src/wp-admin/update-core.php

```diff
@@ -770,6 +777,8 @@ function list_theme_updates() {
 			if ( in_array( $stylesheet, $auto_updates, true ) ) {
 				echo $auto_update_notice;
 			}
+
+			echo $upgrade_notice;
```

Comment:
```suggestion
			echo '<br />' . wp_strip_all_tags( $upgrade_notice );
```

Reply:
Are you also recommending the similar change for plugins? If so I can do that in the other ticket.